3rd-party-tools/slide-tags/spatial-count.jl

```diff
@@ -0,0 +1,561 @@
+using CSV
+using HDF5
+using FASTX
+using ArgParse
+using CodecZlib
+using IterTools: product
+using StatsBase: countmap, sample
+using DataFrames
+using StringViews
+using LinearAlgebra: dot
+using Combinatorics: combinations
+
+# This file was taken from https://github.com/MacoskoLab/Macosko-Pipelines/blob/main/spatial-count/spatial-count.jl
```

Comment:
acknowledge the source of the code explicitly in a "Credits" or "Attributions" section ideally in a separate .md file

Reply:
This script was added here for testing